pkg/printers/internalversion/printers.go

```diff
@@ -435,6 +438,16 @@ func AddHandlers(h printers.PrintHandler) {
 	_ = h.TableHandler(storageClassColumnDefinitions, printStorageClass)
 	_ = h.TableHandler(storageClassColumnDefinitions, printStorageClassList)
 
+	volumeAttributesClassColumnDefinitions := []metav1.TableColumnDefinition{
+		{Name: "Name", Type: "string", Format: "name", Description: metav1.ObjectMeta{}.SwaggerDoc()["name"]},
+		{Name: "DriverName", Type: "string", Description: storagev1alpha1.VolumeAttributesClass{}.SwaggerDoc()["driverName"]},
+		{Name: "Parameters", Type: "string", Description: storagev1alpha1.VolumeAttributesClass{}.SwaggerDoc()["parameters"]},
```

Comment:
Is this for "kubectl get"? If so, then we should not include parameters because that could be very large.

Reply:
@msau42 It prints the numbers of parameters, not parameters‘ contents.

Reply:
I'm not sure how helpful that is for a user. It looks like for StorageClass we don't print anything.

Reply:
@msau42 removed.